### ISSUE TITLE
Don't know how to create ISeq from: clojure.lang.Keyword

### ISSUE THREAD

**Issue by raymond-w-ko** [borkdude/clj-kondo#723]
**version**
clj-kondo v2020.01.27

**platform**
Arch Linux AUR, clj-kondo-bin (I'm guessing this is the GraalVM version.

**editor**
Vim, ALE plugin

**problem**
I noticed that kondo wasn't tracking typos and wasn't working except for unmatched parentheses. I thought I had to manually run it it on the classpath to build a cache. But it crashes with a stack trace.

**repro**
Unfortunately the code is for my company and is proprietary and so it cannot be shared. I will need to see if I have time to extract something useful.

**expected behavior**
Not crash, and at least report what is mis-configured, if anything is. Ideally it should print linting erros.

** stack trace **
```
 > clj-kondo --lint alivesim
Unexpected error. Please report an issue.
java.lang.IllegalArgumentException: Don't know how to create ISeq from: clojure.lang.Keyword
        at clojure.lang.RT.seqFrom(RT.java:557)
        at clojure.lang.RT.seq(RT.java:537)
        at clojure.core$seq__5402.invokeStatic(core.clj:137)
        at clojure.core$reduce1.invokeStatic(core.clj:930)
        at clojure.core$set.invokeStatic(core.clj:4113)
        at clojure.core$set.invoke(core.clj:4105)
        at clj_kondo.impl.utils$map_vals$fn__2719$fn__2720.invoke(utils.clj:205)
        at clojure.lang.PersistentHashMap$NodeSeq.kvreduce(PersistentHashMap.java:1307)
        at clojure.lang.PersistentHashMap$BitmapIndexedNode.kvreduce(PersistentHashMap.java:802)
        at clojure.lang.PersistentHashMap.kvreduce(PersistentHashMap.java:236)
        at clojure.core$fn__8437.invokeStatic(core.clj:6845)
        at clojure.core$fn__8437.invoke(core.clj:6830)
        at clojure.core.protocols$fn__8167$G__8162__8176.invoke(protocols.clj:175)
        at clojure.core$reduce_kv.invokeStatic(core.clj:6856)
        at clj_kondo.impl.utils$reduce_map.invokeStatic(utils.clj:200)
        at clj_kondo.impl.utils$map_vals.invokeStatic(utils.clj:202)
        at clj_kondo.impl.config$fn__3096$delayed_cfg__3097.invoke(config.clj:169)
        at clojure.lang.AFn.applyToHelper(AFn.java:154)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.core$apply.invokeStatic(core.clj:665)
        at clojure.core$memoize$fn__6877.doInvoke(core.clj:6353)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clj_kondo.impl.config$fn__3096$fn__3099.invoke(config.clj:174)
        at clj_kondo.impl.linters$lint_unused_namespaces_BANG_.invokeStatic(linters.clj:396)
        at clj_kondo.core$run_BANG_.invokeStatic(core.clj:97)
        at clj_kondo.main$main.invokeStatic(main.clj:85)
        at clj_kondo.main$main.doInvoke(main.clj:78)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:665)
        at clj_kondo.main$_main$fn__6218.invoke(main.clj:104)
        at clj_kondo.main$_main.invokeStatic(main.clj:104)
        at clj_kondo.main$_main.doInvoke(main.clj:102)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clj_kondo.main.main(Unknown Source)
```

**Comment by borkdude** [comment]
This indicates an error in your .clj-kondo/config.edn. Can you maybe post that here?

**Comment by raymond-w-ko** [comment]
Ugh, I found it. I was copying from the `(ns ...)` form and
forgot to remove the alias `x` in `name.space :as x` in the path ```[:unused-referred-var :exclude]```.

```cljs.core.async :as async [chan close! >!]``` should have been ```cljs.core.async [chan close! >!]```

Sorry about the noise.
